tests/testthat/test-include-gld-purchases.R

```diff
@@ -8,26 +8,32 @@ constants <- tibble::tibble(
 
 lmdb <- tibble::tribble(
   ~atc,
-  "A10abc",
+  "A10A23",
   "A10",
-  "A10123",
+  "A10C",
+  "A10B",
+  "A10AE5",
+  "A10B02",
   "A11",
   "A21",
   "B10A10",
 ) |>
   dplyr::bind_cols(constants)
 
 expected <- tibble::tribble(
-  ~atc,
-  "A10abc",
-  "A10",
-  "A10123",
+  ~atc, ~is_insulin_gld_code, ~is_non_insulin_gld_code,
+  "A10A23", TRUE, FALSE,
+  "A10", FALSE, FALSE,
+  "A10C", FALSE, FALSE,
```

Comment:
@Aastedet you changed these to `NA`, but they can't be `NA`, either true or false. So I switch them to false (since they are neither insulin nor non-insulin drugs, but still GLD drugs (under the `is_gld_code` criteria).